contracts/wallet/MultiSigWallet.sol

```diff
@@ -10,6 +11,7 @@ import "../EIP820/IEIP820Implementer.sol";
 // todo jaycen CAUTION, using eip820 unaudited contracts in multisig inheritance inorder to
 // avoid revert statement otheriwse invoked in the callRecipient function of the tokens mint/send funcs
 contract MultiSigWallet is EIP820Implementer, IEIP820Implementer {
+  using SafeMath for uint256; //todo jaycen PRELAUNCH - make sure we use this EVERYWHERE its needed
```

Comment:
Oh, I missed this guy earlier. Make sure never to change MultiSigWallet or MultiAdmin. Not because this is wrong, but because these are preaudited contracts and we should not ever change them